chainntnfs/neutrinonotify/neutrino.go

```diff
@@ -182,17 +182,15 @@ func (n *NeutrinoNotifier) startNotifier() error {
 		n.txUpdates.Stop()
 		return err
 	}
-	startingHeader, err := n.p2pNode.GetBlockHeader(
-		&startingPoint.Hash,
-	)
+	startingBlock, err := n.GetBlock(startingPoint.Hash)
```

Comment:
Should we perhaps add a small comment here? Something along the lines of "We always fetch the full start block from a peer, but any subsequent blocks are only fetched if filters match"?

Reply:
Will remove this commit and find an alternative, thank you for the time(and sorry for wasting it😢 )!